cmd-info.c

```diff
@@ -769,12 +769,15 @@ void clear_ftrace_info(struct uftrace_info *info)
 
 int command_info(int argc, char *argv[], struct opts *opts)
 {
+	int ret;
 	char buf[PATH_MAX];
 	struct stat statbuf;
 	struct ftrace_file_handle handle;
 	const char *fmt = "# %-20s: %s\n";
 
-	open_data_file(opts, &handle);
+	ret = open_data_file(opts, &handle);
+	if (ret < 0)
+		pr_warn("cannot open data: %s\n", opts->dirname);
```

Comment:
It'd be better adding error string why it failed.  The glibc (+ uClibc and musl) provides "%m" for this.  Note that `pr_err()` does the same thing (but using `strerror_r()`).